Rakefile

```diff
@@ -29,5 +29,4 @@ namespace :debug do
   end
 end
 
-task :test => :compile
-task :default => :test
+task :default => :compile
```

Comment:
Y, I started a different branch for contributing back, with configurable registering of hooks et al. Consider this branch for our use only.